.clang-format

```diff
@@ -22,3 +22,48 @@ PointerAlignment: Left
 Standard: c++14
 TabWidth: 2
 UseTab: Never
+IncludeBlocks: Regroup
+IncludeCategories:
+# Main PCL includes of common module should be sorted at end of PCL includes
+  - Regex:           '^<pcl/[^/]+>$'
+    Priority:        100
+    SortPriority:    101
```

Comment:
Why the counter in 100s (and not 10s)? For dependencies in future? I don't think that'd be a major issue

Based on your logic wrt std headers and preventing future changes, did you consider 190? Result looks nice though (apart from these 2 questions)

Reply:
> Why the counter in 100s (and not 10s)? For dependencies in future? I don't think that'd be a major issue

In case we start to order the modules by dependencies we need more than 10 different free numbers for `SortPriority` - that why I used the base 100 and not 10.
 
> Based on your logic wrt std headers and preventing future changes, did you consider 190? 

What you mean? I have no idea what you want to say 😆